discord/colour.py

```diff
@@ -270,10 +270,15 @@ def darker_grey(cls: Type[CT]) -> CT:
     darker_gray = darker_grey
 
     @classmethod
-    def blurple(cls: Type[CT]) -> CT:
+    def og_blurple(cls: Type[CT]) -> CT:
```

Comment:
```suggestion
    def original_blurple(cls: Type[CT]) -> CT:
```

og is slang

Reply:
"OG Blurple" is how Discord [referred to it officially.](https://blog.discord.com/happy-blurpthday-to-discord-a-place-for-everything-you-can-imagine-fc99ee0a77c0)

> Internally, we first used this new color “blurple_5_plus_adjustments,” but that didn’t quite roll off the tongue as well as Blurple did. The lighter color of Blurple will forever be known as “OG Blurple.”